stacks-common/src/util/secp256k1.rs

```diff
@@ -109,6 +109,11 @@ impl MessageSignature {
             Err(_) => None,
         }
     }
+
+    /// Convert from VRS to RSV
+    pub fn to_rsv(&self) -> Vec<u8> {
```

Comment:
Why is this conversion necessary?

Reply:
Because when recovering or verifying signatures in Clarity, the signature needs to be in RSV order. Whether this helper function is necessary or not is a fine question, it was just helpful to re-use

Reply:
Ugh I just noticed that `secp256k1_recover()` and `secp256k_verify()` don't behave the same as the corresponding methods in `Secp256k1PublicKey` and `MessageSignature`.  The former two methods, which Clarity uses, were added by an itinerant 3rd party contributor who couldn't be bothered to figure out how the existing code worked.

Reply:
Yeah, that's unfortunate because almost every `sign()` implementation returns the signature in RSV, but Clarity uses VRS, so you always have to build a helper method for it. Easy to get tripped up on.